### DISCUSSION TITLE
Opengl

### DISCUSSION THREAD

**Discussion by jbensabat** [alandefreitas/matplotplusplus#170] [Q&A]
Hello
I am trying to use OpenGL
Is there any guideline on how to do so for example on a polar plot ? 
thanks
Jac

**Comment by alandefreitas** [comment]
Hi @jbensabat ,

The OpenGL backend is experimental. It doesn't work for polar plots yet.

**Comment by jbensabat** [comment]
Hi
Thanks for the info
OpenGL could be a great platform for this lib
Best
Jac

Sent from my iPhone

On 21 Jun 2021, at 21:13, Alan de Freitas ***@***.***> wrote:

﻿

Hi @jbensabat<https://github.com/jbensabat> ,

The OpenGL backend is experimental. It doesn't work for polar plots yet.
